sidebars.js

```diff
@@ -163,7 +163,7 @@ const sidebars = {
                     'id': 'basics/user-interface/controls/creating-controls/index'
                   },
                   'items': [
-                    'basics/user-interface/controls/creating-controls/choosing-a-custom-contro-type',
+                    'basics/user-interface/controls/creating-controls/choosing-a-custom-control-type',
```

Comment:
If any link was changed, we should also update redirects file. Just to keep old links work fine:
https://github.com/AvaloniaUI/avalonia-docs/blob/b88a7ee4ff05bd6b7f8699d198844ff47bada1a3/redirects.js

Reply:
Ah, yes, good catch

Reply:
Should be fixed

Reply:
According to the test failure the link I redirected to was invalid:

```
        {
            from: '/docs/basics/user-interface/controls/creating-controls/choosing-a-custom-contro-type',
            to: '/docs/basics/user-interface/controls/creating-controls/choosing-a-custom-control-type',
        },
```

Is it correct to include 'next' in the path like `/docs/next/basics`? If so, I'm not sure how the other links work. None of the others had it.

Reply:
Other redirects were for the old documentation (i.e. 0.10). 
11.0 documentation still uses "next" prefix, so it should be included, yes.

Although, ideally we should remove this prefix and adjust all of these redirects.

Reply:
I see. Yes I think the current docs need to remove the 'next' prefix... which would create a redirect for everything.

The 0.10.x docs are the ones that need a version if anything. So it's setup backwards of what I would expect.

Reply:
Should be fixed, let's see if tests pass.